getting-started/gopher-run/cmd/main.go

```diff
@@ -114,34 +112,12 @@ unroll,0,0,0,0,0,0,0,0,0,0,0`)
 			log.Printf("ioutil.ReadAll: %v", err)
 			continue
 		}
-		data = append(data, b...)
+		data = append(append(data, []byte(`
+`)...), b...)
```

Comment:
Separate calls? Also, if it only needs a newline, probably better to do `"\n"`.

Reply:
Done